gulp.config.js

```diff
@@ -81,12 +42,109 @@ module.exports = function() {
             // Seems to break sourcemaps, so conditionally minify css in gulpfile
             // outputStyle: 'compressed',
             precision: 8,
+            includePaths: [src + theme, src + base]
         },
         uglifySettings: {
             mangle: false,
         },
     };
 
+    setConfigPaths();
     return config;
+
+    //////////////////////////////////
+
+    // Fill the config with themes paths
+    function setConfigPaths() {
+        addThemePaths('./' + src + config.base + '/');
+        addThemePaths('./' + src + config.theme + '/');
+
+        // Styles root
+        var themeStylesRoot = './' + src + config.theme + '/app.scss';
+        var match = glob.sync(themeStylesRoot);
+        config.styles.root = (match.length) ? themeStylesRoot : './' + src + config.base + '/app.scss';
+
+        return config;
+    }
+
+    // Adds paths of a single theme to the config
+    function addThemePaths(themeDir) {
+        // Theme paths patterns
+        var themePaths = {
```

Comment:
Are the following theme paths required for every theme, or are expected, but it is fine if they are not present?

I'm asking this because we are going to need to document this for theme creators by the time we announce the free version of Ottemo.